src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -50,25 +48,24 @@ const SetAclPermsDocContainerForm = () => {
     const permissions = event.target.setAclPerms.value
       ? {
           read: event.target.setAclPerms.value === 'Give',
-          write: event.target.setAclPerms.value === 'Give',
           append: event.target.setAclPerms.value === 'Give'
         }
       : undefined;
-    let podUsername = event.target.setAclTo.value;
+    let otherPodUsername = event.target.setAclTo.value;
 
-    if (!podUsername) {
-      podUsername = selectedUser;
+    if (!otherPodUsername) {
+      otherPodUsername = selectedUser.username;
     }
 
-    if (!podUsername) {
+    if (!otherPodUsername) {
       runNotification('Set permissions failed. Reason: Username not provided.', 5, state, dispatch);
       setTimeout(() => {
         clearInputFields();
       }, 3000);
       return;
     }
 
-    if (getPodUrl(podUsername) === podUrl) {
+    if (getPodUrl(otherPodUsername) === podUrl) {
```

Comment:
Would likely change to use podUrl explicitly for the input field in the future instead of using username to get a podUrl.

The function getPodUrl assumes a specific pod URL structure (i.e. http://pod.example.com/). Would completely break down if pod URL is instead "http://example.com/pod/". Will open issue for that.